batch7crud-roll6/src/main/java/com/lftechnology/batch7crud/controller/UserController.java

```diff
@@ -0,0 +1,214 @@
+package com.lftechnology.batch7crud.controller;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.User;
+import com.lftechnology.batch7crud.service.UserService;
+import com.lftechnology.batch7crud.util.TypeCaster;
+
+/**
+ * @author madandhungana <madandhungana@lftechnology.com> Jan 18, 2016
+ */
+
+@WebServlet("/users/*")
+public class UserController extends HttpServlet {
+
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = Logger.getLogger("UserController");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)